06-exceptions-mandatory-tasks/src/com/fsher/exercise4/Exercise4.java

```diff
@@ -0,0 +1,40 @@
+package com.fsher.exercise4;
+
+import com.fsher.exercise4.exceptions.NumberZeroException;
+import com.fsher.exercise4.exceptions.TooLargeNumberException;
+import com.fsher.exercise4.exceptions.TooLowNumberException;
+
+import java.util.Scanner;
+
+public class Exercise4 {
+    private static final int MAX = 466;
+    private static final int MIN = -23;
+
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+
+        System.out.print("Enter any number: ");
+        try (scanner) {
+            int number = scanner.nextInt();
+            System.out.printf("Here is your number: %d \n", Exercise4.checkNumber(number));
+        } catch (Exception e) {
```

Comment:
This will catch not only your custom exceptions but all of the potential exceptions.
The correct way to catch only yours in a single catch is `catch (TooLargeNumberException | TooLowNumberException | NumberZeroException e)`